spec/services/idv/steps/in_person/state_id_step_spec.rb

```diff
@@ -239,4 +239,126 @@
       end
     end
   end
+
+  describe '#call' do
+    let(:pii_from_user) { flow.flow_session[:pii_from_user] }
+    let(:params) { ActionController::Parameters.new({ state_id: submitted_values }) }
+    let(:capture_secondary_id_enabled) { true }
+    let(:dob) { '1972-02-23' }
```

Comment:
for lines 247-260 lets use the [constants](https://github.com/18F/identity-idp/blob/main/spec/support/features/in_person_helper.rb#L13-L32) in the `in_person_helper` file